model/model.go

```diff
@@ -4,44 +4,46 @@ import (
 	"sync"
 )
 
+// CallBack represents a callback object with the necessary fields
 type CallBack struct {
 	HttpMethod  string `form:"httpMethod" json:"httpMethod" validate:"required"`
 	CallbackUrl string `form:"callbackUrl" json:"callbackUrl" validate:"required"`
 	PublicKey   string `form:"key" json:"key"`
 }
 
+// Subscriber represents a subscriber object with a callback and channel
 type Subscriber struct {
 	CallBack CallBack `form:"callback" json:"callback" validate:"required"`
 	Channel  string   `form:"channel" json:"channel" validate:"required"`
 }
 
+// Publisher represents a publisher object with a unique id and channel
```

Comment:
What is publisher for? What is the id for? The comments should explain the purpose of the fields.

Reply:
done